assets/javascripts/discourse-assign/initializers/extend-for-assigns.js.es6

```diff
@@ -318,6 +320,57 @@ export default {
     if (!siteSettings.assign_enabled) {
       return;
     }
+    const currentUser = container.lookup("current-user:main");
+    if (currentUser.can_assign) {
+      const REGEXP_USERNAME_PREFIX = /^(user:|@)/gi;
```

Comment:
This filter seems copied from the normal user selector (where you can use `@david` or `user:david`). Do we need it here?

Reply:
yaa ill change the regex